test/index.test.js

```diff
@@ -334,5 +356,68 @@ describe('module-resolver', () => {
                 transformerOpts,
             );
     });
+
+    describe('unknown filename', () => {
+      const unknownFileTransformerOpts = {
+        babelrc: false,
+        plugins: [
+          [plugin, {
+            root: [
+              './src',
+            ],
+            cwd: 'babelrc',
+          }],
+        ],
+      };
+      const cachedCwd = process.cwd();
+      const babelRcDir = 'test/testproject';
+
+      beforeEach(() => {
+        process.chdir(babelRcDir);
+      });
+
+      afterEach(() => {
+        process.chdir(cachedCwd);
+      });
+
+      describe('should resolve the sub file path', () => {
+        testRequireImport(
+                  'components/Root',
+                  './src/components/Root',
+                  unknownFileTransformerOpts,
+              );
+      });
+    });
+
+    describe('missing babelrc in path (uses cwd)', () => {
+      jest.mock('find-babel-config', () => ({
+        sync: function findBabelConfigSync() {
+          return { file: null, config: null };
+        },
+      }));
+      jest.resetModules();
+      const pluginWithMock = require.requireActual('../src').default;
+
+      const missingBabelConfigTransformerOpts = {
+        babelrc: false,
+        plugins: [
+          [pluginWithMock, {
+            root: [
+              '.',
+            ],
+            cwd: 'babelrc',
+          }],
+        ],
+        filename: './test/testproject/src',
```

Comment:
The filename is incorrect, and thus the result is incorrect.
The result should only be `./components/Root` I believe.